code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -455,8 +455,8 @@
 		if(istype(target, /obj/item/clothing/suit/space/hardsuit/mining) || istype(target, /obj/item/clothing/head/helmet/space/hardsuit/mining))
 			var/obj/item/clothing/C = target
 			var/list/current_armor = C.armor
-			if(current_armor.["melee"] < 80)
-				current_armor.["melee"] = min(current_armor.["melee"] + 10, 80)
+			if(current_armor.["melee"] < 50)
+				current_armor.["melee"] = min(current_armor.["melee"] + 10, 50)
```

Comment:
I wouldnt mind spreading the values around to bullet/laser if youre concerned with pvp. I just dont want the pve to be boring/done after you farm a tendril